### DISCUSSION TITLE
Wsparcie dla GSM/GPRS

### DISCUSSION THREAD

**Discussion by wparuch** [hackerspace-silesia/Smogomierz#135] [Ideas]
Czy jest szansa na rozszerzenie projektu o pakietową transmisję danych? Byłoby to bardzo przydatne w miejscach gdzie nie ma wifi, a jest zanieczyszczone powietrze :). Np przy użyciu modułu SIM800 / SIM900 / SIM908 / A6 / SIM5320 / SIM5360 / SIM7100 ale to bardzo luźne propozycje.

**Comment by firemark** [comment]
Cześć - jak jesteś z okolic Śląska to możemy się omówić i sami napisać obsługę i sposób podłączenia do esp :)

**Comment by wparuch** [comment]
Niestety, małopolskie

**Comment by bfaliszek** [comment]
Ciekawy pomysł. Myślę też o możliwości zapisu pomiarów w pamięci ESP lub na karcie pamięci i wysyłaniu ich ich dopiero, gdy będzie dostęp do sieci lub co kilkadziesiąt minut. 
Jeśli chodzi o transmisję danych GSM/GPRS to nie powinno być większego problemu. Tylko tu chyba też byłoby lepiej wysyłać zbiorczo pomiary raz na kilkadziesiąt minut, niż np. co minutę czy tam pięć.

**Comment by wparuch** [comment]
Cache'owanie danych to też fajny pomysł, historia byłaby ciągła.

Aczkolwiek przy GPRSie osobiście wolałbym mieć odczyt +- na żywo, analogiczny jak po WiFi. Można zrobić na to flagę wg preferencji usera.

**Comment by bfaliszek** [comment]
Jakiś czas temu, znajomy podrzucił mi pomysł montowania Smogomierza na dronie, dlatego zacząłem się zastanawiać nad jakimś cache'owaniem danych i wysyłaniem ich dopiero przy dostępnie do WiFi. GPRS mógłby pozwolić na dostęp do tych danych na bierząco. Tylko tutaj problemem chyba będzie waga baterii, która musiałaby to wszystko zasilać i jeszcze zmieścić się na dronie ;)

**Comment by wparuch** [comment]
Niezłe! Myślę, że na kilkunastominutowe loty (bo na więcej zapewne bateria drona nie pozwoli) ciężar baterii LiPo byłby to do przeskoczenia. Do tego interfejs do ładowania przez USB (np z ładowarki samochodowej) + moduł gps i jest zestaw kompletny

**Comment by wiuwiu** [comment]
Dron DJI Phantom dałby radę unieść taką konstrukcję. Swego czasu miałem taką konstrukcję Ardunio z modułem GSM i podpiętym czujnikiem pyłów. Całość zasilana z dołączonej baterii. [https://www.dfrobot.com/index.php?route=product/product&product_id=1448&search=dfr0355&description=true&category_id=48#.VzrutOT2HHY](url)
